graylog2-web-interface/src/stores/streams/StreamRulesStore.ts

```diff
@@ -75,6 +76,9 @@ class StreamRulesStore {
   _emitChange() {
     this.callbacks.forEach((callback) => callback());
   }
+  unregister(callback: Callback) {
+    lodash.pull(this.callbacks, callback);
```

Comment:
I really appreciate the thought you are putting into this, thank you. I have a differing opinion towards lodash though, because it's not like react/reflux something you need to learn to be able to write code, but it is something you can utilize when you see the need for it. So it does not raise the barrier for new team members to be productive. Instead, it allows to write more concise and succinct code when you are ready.

Long story short, I am gonna add the explicit dependency and I hope you are alright with that.